Dockerfile

```diff
@@ -47,15 +48,7 @@ ENV STORAGE_LOCAL_ROOT="/directus/uploads"
 ENV NODE_ENV="production"
 ENV NPM_CONFIG_UPDATE_NOTIFIER="false"
 
-COPY --from=pruned /workspace/pruned/cli.js cli.js
-COPY --from=pruned /workspace/pruned/dist dist
-COPY --from=pruned /workspace/pruned/package.json package.json
-COPY --from=pruned /workspace/pruned/node_modules node_modules
-
-RUN mkdir /directus/database /directus/extensions /directus/uploads \
-	&& chown -R node:node /directus \
-	&& corepack enable \
-	&& corepack prepare pnpm@7.30.0 --activate
+COPY --from=pruned --chown=node:node /workspace/pruned .
```

Comment:
The `pruned` folder already contains just the build essentials, so we don't have to manually chose individual files

Reply:
Love it ❤️🏅